docs/cookbook.md

```diff
@@ -123,13 +123,16 @@ Sort a list of ISO 8601 date/time strings, for example to place log entries in o
 
 ### Round a time down to whole hours
 
-Use the `with()` method of each Temporal type if you want to round or balance the fields.
+Use the `with()` method of each calendar-based Temporal type if you want to round or balance the fields.
```

Comment:
Time isn't calendar based, but does have a `with` method. Should this be "non-`Absolute`" instead of "calendar-based" ?

Reply:
You're right, this is a confusing shorthand — I really wish there was a word for "calendar and/or wall-clock based"! "Civil" would do nicely except it was decided long ago that that brought in its own confusion.